rewrite-maven/src/main/java/org/openrewrite/maven/internal/MavenPomDownloader.java

```diff
@@ -602,8 +602,9 @@ public Pom download(GroupArtifactVersion gav,
                             pom = pom.withGav(pom.getGav().withDatedSnapshotVersion(versionMaybeDatedSnapshot));
                         }
                         mavenCache.putPom(resolvedGav, pom);
-                        ctx.getResolutionListener().downloadSuccess(resolvedGav, containingPom);
-                        sample.stop(timer.tags("outcome", "downloaded").register(Metrics.globalRegistry));
+                        long nanos = sample.stop(timer.tags("outcome", "downloaded").register(Metrics.globalRegistry));
+                        ctx.getResolutionListener()
+                                .downloadSuccess(resolvedGav, containingPom, Duration.ofNanos(nanos));
```

Comment:
So the old, durationless event type still exists but will now never be called? That isn't great. If the old event is still going to exist it should continue to be called.

Reply:
I would prefer to remove the old event under the assumption only Moderne is consumers of it.